src/Collisions/babylon.collisionCoordinator.ts

```diff
@@ -390,7 +392,8 @@ module BABYLON {
             // Check all meshes
             for (var index = 0; index < this._scene.meshes.length; index++) {
                 var mesh = this._scene.meshes[index];
-                if (mesh.isEnabled() && mesh.checkCollisions && mesh.subMeshes && mesh !== excludedMesh) {
+                if(!mesh._collisionGroup) mesh._collisionGroup = -1;

```

Comment:
that would break the freeCamera through, i added the collisionMask of the freeCamera to the collider, since it doesn't seem like the camera is passed further into the collision code, i'll remove the check completely and it should be okay.